apps/coa/fields/index.js

```diff
@@ -327,6 +327,35 @@ module.exports = {
     formatter: ['ukPostcode'],
     className: ['govuk-input', 'govuk-input--width-10']
   },
+  'legal-company-name': {
+    validate: ['notUrl'],
```

Comment:
Maybe legal-company-name should have a maxlength validator?

Reply:
Added maxlength validation for company name to allow upto 200 characters